src/main/webapp/assets/js/feedScript.js

```diff
@@ -639,10 +651,31 @@ async function submitModal() {
  */
 window.onclick = function(event) {
   if (modal && event.target == modal) {
-    closeModal();
+    modal.style.display = 'none';
+    createPostButton.focus();
   }
 };
 
+document.addEventListener('keydown', function(e) {
+  const isTabPressed = e.key === 'Tab' || e.keyCode === 9;
+
+  if (!isTabPressed) {
+    return;
+  }
+  // If user is trying to go to the previous element, make sure it wraps to the bottom
```

Comment:
End with period.

Reply:
Same for comment below. Start with capital.

Reply:
Thanks!